site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -171,7 +187,17 @@
                         </td>
                         <td> {{ poll.getReleaseDate() }} </td>
                         <td id="poll_{{ poll.getId() }}_responses"> {{ poll.getUserResponses()|length }} </td>
-                        <td><a href="{{base_url}}/viewResults/{{ poll.getId() }}" class="btn btn-primary">View Results</a></td>
+                        <td>
+                            <a href="{{base_url}}/viewResults/{{ poll.getId() }}" class="btn btn-primary">
+                                {% if poll.isStudentHistogramAvailable() %}
+                                    <i class="fas fa-lock-open">
+                                {% else %}
+                                    <i class="fas fa-lock">
+                                {% endif %}
+                                    &nbsp;View Results
```

Comment:
I think you missed the fix for the font in the future polls table, but the fix did work on the other tables :)